osgtest/tests/test_450_xrootd.py

```diff
@@ -33,18 +33,18 @@ def test_01_xrdcp_local_to_server(self):
         fail = core.diagnose('xrdcp copy, local to URL',
                              command, status, stdout, stderr)
         file_copied = os.path.exists(os.path.join(temp_dir, 'copied_file.txt'))
-        if core.config['xrootd.multiuser'] != "ON":
-            shutil.rmtree(temp_dir)
+        if file_copied:
+            core.state['xrootd.copied-to-server'] = True
         self.assertEqual(status, 0, fail)
         self.assert_(file_copied, 'Copied file missing')
 
     def test_02_xrootd_multiuser(self):
-        core.skip_ok_unless_installed('xrootd', 'xrootd-client', 'xrootd-multiuser', by_dependency=True)
-        temp_dir = "/tmp/vdttest"
-        if core.config['xrootd.multiuser'] == "ON":
+        core.skip_ok_unless_installed('xrootd', 'xrootd-client', 'globus-proxy-utils', 'xrootd-multiuser', by_dependency=True)
+        self.skip_bad_unless(core.state['xrootd.copied-to-server'] is True, 'File to check ownership does not exist')
+        temp_dir = core.config['xrootd.tmp-dir']
```

Comment:
You only use `temp_dir` once in this function, you may as well get rid of this line and replace `temp_dir` with `core.config['xrootd.tmp-dir']`